src/main/utils/Settings.js

```diff
@@ -8,15 +9,21 @@ const environment = process.env;
 
 class Settings {
   constructor(jsonPrefix, wipeOldData) {
-    const DIR = (environment.APPDATA ||
+    const DIR = app.getPath('userData');
+    this.PATH = `${DIR}/${(jsonPrefix || '')}.settings.json`;
+    const OLD_DIR = (environment.APPDATA ||
       (process.platform === 'darwin' ? environment.HOME + '/Library/Preferences' : os.homedir())) +
       '/GPMDP_STORE';
-    this.PATH = `${DIR}/${(jsonPrefix || '')}.settings.json`;
+    const OLD_PATH = `${OLD_DIR}/${(jsonPrefix || '')}.settings.json`;
     this.data = initalSettings;
     this.lastSync = 0;
 
     if (fs.existsSync(this.PATH) && !wipeOldData) {
       this._load();
+    } else if (fs.existsSync(OLD_PATH)) {
+      fs.renameSync(OLD_PATH, this.PATH);
+      fs.rmdir(OLD_DIR, function () {});
```

Comment:
Can you confirm your logic for the asyncronous usage of rmdir here?

I don't think it's a problem and will actually be better performance :+1: 

Was it deliberate?